app/src/main/java/de/tum/in/tumcampusapp/managers/TransportManager.java

```diff
@@ -148,9 +148,8 @@ public TransportManager(Context context) {
      * @return True, if favorite
      */
     public boolean isFavorite(String symbol) {
-        try (Cursor c = db.rawQuery("SELECT * FROM transport_favorites WHERE symbol = ?", new String[]{symbol})) {
-            return c.getCount() > 0;
-        }
+        List<TransportFavorites> favorites = transportDao.getFavorites(symbol);
+        return favorites.size() > 0;
```

Comment:
This logic should be handled completely in the DAO.
I think you could write a query like: 
```SQL
SELECT EXISTS(SELECT * FROM transport_favorites WHERE symbol = :symbol);
```
directly returning a boolean